### DISCUSSION TITLE
Busco equipo :)

### DISCUSSION THREAD

**Discussion by denissegutierrez** [Launch-X-Latam/MisionFrontEnd#569] [Team Posts]
Si a alguien le falta un integrante este es mi correo lx00689@innovaccion.mx

**Comment by loo-kuhs** [comment]
Hola @denissegutierrez ✌, **yo busco equipo**. Me gustaría integrarme al tuyo si no hay problema. 
Mi correo es lx04129@innovaccion.mx

**Comment by JaquelineJT** [comment]
Hola me interesa unirme a su equipo, mi correo es lx02054@innovaccion.mx

**Comment by denissegutierrez** [comment]
Claro, comparteme tu correo y ahora veo como hacer un grupo

**Comment by jriveramdev** [comment]
¿@denissegutierrez aún tienen cupo en el equipo?
Quiero integrarme a un equipo, mi correo es lx14856@innovaccion.mx si aún tienen lugar, gracias.

**Comment by denissegutierrez** [comment]
claro ahora te agrego

**Comment by danielesp18** [comment]
Hola👋
Aun alcanza uno más👀??
Mi correo es lx15787@innovaccion.mx por si aun se puede.
Gracias.

**Comment by denissegutierrez** [comment]
Ahora te agrego, porfavor confiermen que estan bien en el equipo o si ya tienen otro, con un comentario en teams

**Comment by Alejandro-Ari** [comment]
Hola aún hay lugar para uno más ??

**Comment by Alejandro-Ari** [comment]
lx00580@innovaccion.mx

**Comment by cesar-ayar** [comment]
lx04190@innovaccion.mx

**Comment by HCZ369** [comment]
Hola! si aun les queda cupo yo todavia no tengo equipo! :( lx09008@innovaccion.mx

**Comment by denissegutierrez** [comment]
Ya no nos quedan mas espacios
